.github/workflows/ci.yml

```diff
@@ -10,7 +10,6 @@ jobs:
     name: build
     runs-on: ubuntu-latest
     strategy:
-      fail-fast: true
```

Comment:
Out of curiosity: Why remove this? Is it because of the intermittent/flaky failures?

Reply:
Yes, 1.3 CI often needs to be run multiple times. This will fail all jobs in the matrix if one fails. I removed this to prevent that from happening.